moped-editor/src/views/projects/projectView/ProjectTeamTable.js

```diff
@@ -371,12 +399,25 @@ const ProjectTeamTable = ({
                 return personnel[item];
               })
         }
-        title="Project team"
+        title={
+          <Typography variant="h2" color="primary">
+            Project team
+          </Typography>
+        }
```

Comment:
Whoa, another one. I would have never imagined you could just pass your own component. 💯